draft-irtf-cfrg-frost.md

```diff
@@ -464,19 +445,19 @@ structures into values that can be processed with hash functions.
 
 ~~~
   Inputs:
-  - commitment_list = [(j, D_j, E_j), ...], a list of commitments issued by each signer,
+  - commitment_list = [(i, x, y), ...], a list of commitments issued by each signer,
     where each element in the list indicates the signer index and their
-    two commitment Element values. B MUST be sorted in ascending order
+    two commitment Element values (x, y). This list MUST be sorted in ascending order
     by signer index.
 
-  Outputs: A byte string containing the serialized representation of B.
+  Outputs: A byte string containing the serialized representation of commitment_list.
 
   def encode_group_commitment_list(commitment_list):
     encoded_group_commitment = nil
-    for (j, D_j, E_j) in B:
-      encoded_commitment = I2OSP(j, 2) ||
-                           G.SerializeElement(D_j) ||
-                           G.SerializeElement(E_j)
+    for (index, hiding_nonce_commitment, blinding_nonce_commitment) in commitment_list:
```

Comment:
If you are feeling up for it, `participant_index` is more accurate, as opposed to just the ordering of the list.

Reply:
`index` is the notation used in the parameter description, so I'll leave this as-is for now.